cmd/svcat/class/create_cmd.go

```diff
@@ -21,24 +21,29 @@ import (
 
 	"github.com/kubernetes-incubator/service-catalog/cmd/svcat/command"
 	"github.com/kubernetes-incubator/service-catalog/cmd/svcat/output"
+	servicecatalog "github.com/kubernetes-incubator/service-catalog/pkg/svcat/service-catalog"
 	"github.com/spf13/cobra"
 )
 
 // CreateCmd contains the information needed to create a new class
 type CreateCmd struct {
 	*command.Context
-	Name string
-	From string
+	Name      string
+	From      string
+	Namespace string
```

Comment:
@jberkhahn is right, this should be using the `command.Namespaced` and `command.Scoped` structs, see 

https://github.com/kubernetes-incubator/service-catalog/blob/master/cmd/svcat/broker/get_cmd.go

for an example of how that affects the command (mostly you don't have a field for Namespace or Cluster on the command), and then hook them up by calling `createCmd.AddNamespaceFlags(cmd.Flags(), false)`, etc.